components/inputs/input-date.js

```diff
@@ -57,50 +71,150 @@ class InputDate extends LocalizeStaticMixin(RtlMixin(LitElement)) {
-			'ja': { chooseDate: 'Choose Date' },
-			'ko': { chooseDate: 'Choose Date' },
-			'nl': { chooseDate: 'Choose Date' },
-			'pt': { chooseDate: 'Choose Date' },
-			'sv': { chooseDate: 'Choose Date' },
-			'tr': { chooseDate: 'Choose Date' },
-			'zh': { chooseDate: 'Choose Date' },
-			'zh-tw': { chooseDate: 'Choose Date' }
+			'ar': {
+				clear: 'مسح',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today',
+			},
+			'da': {
+				clear: 'Ryd',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today',
+			},
+			'de': {
+				clear: 'Löschen',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'en': {
+				clear: 'Clear',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'es': {
+				clear: 'Borrar',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'fr': {
+				clear: 'Effacer',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'ja': {
+				clear: 'クリア',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'ko': {
+				clear: '지우기',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'nl': {
+				clear: 'Wissen',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'pt': {
+				clear: 'Desmarcar',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'sv': {
+				clear: 'Rensa',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'tr': {
+				clear: 'Temizle',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'zh': {
+				clear: '清除',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			},
+			'zh-tw': {
+				clear: '清除',
+				openInstructions: 'Arrow down or press enter to access mini-calendar',
+				setToToday: 'Set to Today'
+			}
 		};
 	}
 
+	constructor() {
+		super();
+
+		this._dropdownOpened = false;
+		this._formattedValue = '';
+
+		getCalendarDescriptor();
+	}
+
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
-		this._formattedValue = this.value ? formatDate(parseISODate(this.value)) : '';
+		this._dropdown = this.shadowRoot.querySelector('d2l-dropdown-content');
 	}
 
 	render() {
-		const placeholder = this.placeholder || this.localize('chooseDate');
+		const placeholder = this.placeholder || calendarDataDescriptor.formats.dateFormats.short;
```

Comment:
I changed the placeholder to be the accepted short date format (this was included as an option in the design) since at the moment the format that will be accepted by our parser is fairly limited.

Reply:
A couple things:
- This makes me question whether we should support a custom placeholder at all... it seems like the format hint is super important, and if in some places it gets replaced by a custom placeholder, then those users don't get it.
- Along the same lines, the placeholder doesn't get read to screen readers, so they're not going to get the hint. Maybe an offscreen `aria-describedby` that says "Format: YYYY-MM-DD" might help?

Reply:
Agreed, that does seem problematic. Perhaps we shouldn't include it initially and see if use cases pop up requesting it, then evaluate further.

I have found that the placeholder does get read (at least with NVDA - will confirm voice over). I like the idea of an `aria-describedby` too though in case there is a value there just to make format extra clear.

Reply:
Yeah I'd be on board with removing it entirely assuming Jeff is good with it.

Reply:
Confirmed with Jeff that this would be okay.